src/resources/locales/de.json

```diff
@@ -151,6 +151,10 @@
             "bookmarkTitle": "bookmark",
             "detachWindowTitle": "das Fenster lösen",
             "fullscreenTitle": "fullscreen mode",
+            "goTo": "Go to",
+            "goToError": "This page does not exist",
+            "goToPlaceHolder": "Go to page",
```

Comment:
I think that the text input field "placeholder" should be something like "enter a page number", not a description of the associated action.